tests/plugins/autogrow/autogrow.js

```diff
@@ -56,6 +56,25 @@
 
 				editor.execCommand( 'autogrow' );
 
+				wait();
+			} );
+		},
+
+		// (#4891)
+		'test autogrow shouldn\'t add inline width property to editor container': function() {
+			var editor = this.editor,
+				bot = this.editorBot;
+
+			bot.setData( autogrowTools.getTestContent( 10 ), function() {
+				editor.once( 'afterCommandExec', function() {
+					resume( function() {
+						var editorContainerInlineWidth = editor.container.$.style.width;
```

Comment:
Is it possible to use our own API to receive the `width` style? Like: https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_dom_element.html#method-getStyle

Sometimes those methods have some small differences, but we should stick to the API whenever possible.